### ISSUE TITLE
Threw an error when scrolling terminal

### ISSUE THREAD

**Issue by sxzz** [webfansplz/vtui-repl#2]
```
/Users/kevin/Library/pnpm/global/5/.pnpm/yoga-layout-prebuilt@1.10.0/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js:53
        throw ex;
        ^

TypeError: Cannot read properties of undefined (reading 'key')
    at file:///Users/kevin/Library/pnpm/global/5/.pnpm/vtui-repl@0.0.3_typescript@4.8.4/node_modules/vtui-repl/dist/main.mjs:6:1793
    at file:///Users/kevin/Library/pnpm/global/5/.pnpm/vue-termui@0.0.17_@vue+runtime-core@3.2.41/node_modules/vue-termui/dist/index.bff876d6.js:1003:9
    at Set.forEach (<anonymous>)
    at ReadStream.s (file:///Users/kevin/Library/pnpm/global/5/.pnpm/vue-termui@0.0.17_@vue+runtime-core@3.2.41/node_modules/vue-termui/dist/index.bff876d6.js:1002:14)
    at ReadStream.emit (node:events:525:35)
    at addChunk (node:internal/streams/readable:324:12)
    at readableAddChunk (node:internal/streams/readable:293:11)
    at Readable.push (node:internal/streams/readable:234:10)
    at TTY.onStreamRead (node:internal/stream_base_commons:190:23)

```

**Comment by sxzz** [comment]
https://user-images.githubusercontent.com/6481596/205078597-b7548e8b-ac1a-4be8-bd6b-95ebc8f801d2.mp4

Reproduce

**Comment by webfansplz** [comment]
Fixed in v0.0.4
